styles/variables.less

```diff
@@ -162,8 +162,8 @@
 @sand-picker-font-size:             54px;
 @sand-picker-joined-font-size:      66px;
 @sand-steps-step-number-font-size:  54px;
-@sand-steps-step-dot-font-size:     30px;
-@sand-steps-step-current-dot-font-size:  48px;
+@sand-steps-step-highlightCurrent-font-size:         24px;
```

Comment:
I guess this can be replaced by `size` prop in Steps.

Reply:
As I mentioned above, I think we need to modified the font size of the icon.